code/game/gamemodes/game_mode.dm

```diff
@@ -541,3 +651,83 @@
 	for(var/V in station_goals)
 		var/datum/station_goal/G = V
 		G.print_result()
+
+/datum/game_mode/proc/generate_credit_text()
+		var/list/round_credits = list()
+		var/len_before_addition
+
+		// HEADS OF STAFF
+		round_credits += "<center><h1>The Glorious Command Staff:</h1>"
+		len_before_addition = round_credits.len
+		for(var/datum/mind/current in SSticker.mode.get_all_heads())
+				round_credits += "<center><h2>[current.name] as the [current.assigned_role]</h2>"
+		if(round_credits.len == len_before_addition)
+				round_credits += list("<center><h2>A serious bureaucratic error has occurred!</h2>", "<center><h2>No one was in charge of the crew!</h2>")
+		round_credits += "<br>"
+
+		// SILICONS
+		round_credits += "<center><h1>The Silicon \"Intelligences\":</h1>"
+		len_before_addition = round_credits.len
+		for(var/datum/mind/current in SSticker.mode.get_all_silicon())
+				round_credits += "<center><h2>[current.name] as the [current.assigned_role]</h2>"
+		if(round_credits.len == len_before_addition)
+				round_credits += list("<center><h2>[station_name()] had no silicon helpers!</h2>", "<center><h2>Not a single door was opened today!</h2>")
+		round_credits += "<br>"
+
+		// SECURITY
+		round_credits += "<center><h1>The Brave Security Officers:</h1>"

```

Comment:
Call it `Security Staff` instead. Not everyone is an officer.

Reply:
I'd keep it officers, they are still all officers of the law regardless of if they are hos or warden

Reply:
I think officers is fine as well.